### PR TITLE
Refactor mount-helper-container

### PR DESCRIPTION
**Pull request by prankulmahajan**
This PR includes:

1. Add the rpc calls required for the file driver
2. Make targets required for package building
3. UT+ coverage

### CODE REVIEW

File: mount-helper-container/Makefile [hunk 1]
```diff
@@ -16,18 +32,98 @@ BIN_DIR := bin
...
+	rpmbuild -ba --build-in-place --define "_topdir $(PWD)/$(BUILD_DIR)/rpm" $(REDHAT_SPEC)
+	cp  $(BUILD_DIR)/rpm/RPMS/* ./
+	rm -rf $(BUILD_DIR)
+
+packages:
+packages: deb-build rpm-build clean
+
 build:
-	CGO_ENABLED=0 GOOS=$(go env GOOS) GOARCH=$(go env GOARCH) go build -mod mod -o ${BIN_DIR}/mhelper-container-server ./server/server.go
+	CGO_ENABLED=0 GOOS=$(go env GOOS) GOARCH=$(go env GOARCH) go build -mod mod -o ${BIN_DIR}/eitserver ./server/server.go
```
Comment (saicharansunkara-ibm) [inline mount-helper-container/Makefile:107]: Any specific reason why this has been renamed from mhelper-container-server to eitserver? mhelper-container-server sounds specific to MH code while eitserver sounds more generic.

**Review by saicharansunkara-ibm — COMMENTED** [review]
Had a question why directory name is changed.

File: mount-helper-container/Makefile (hunk 1 above)
Comment (prankulmahajan) [reply mount-helper-container/Makefile:107]: No particular reason. I can change that

Comment (prankulmahajan) [reply mount-helper-container/Makefile]: Done

**Comment by prankulmahajan** [comment]
@saicharansunkara-ibm Can you please take a look again? Thanks

**Review by saicharansunkara-ibm — APPROVED** [review]

**Review by anand-desai — APPROVED** [review]

**Review by arahamad — COMMENTED** [review]
we need to add support to create package via workspace

**Pull request merged**